cmd/lookup_cmd.go

```diff
@@ -47,14 +50,17 @@ rbac-tool lookup -ne '^system:.*'
 			var re *regexp.Regexp
 			var err error
 
-			if regex != "" {
-				re, err = regexp.Compile(regex)
-			} else {
-				if len(args) != 1 {
-					re, err = regexp.Compile(fmt.Sprintf(`.*`))
+			if regex == "" {
+				if len(args) > 0 {
```

Comment:
args should be exactly one

Reply:
Done: https://github.com/alcideio/rbac-tool/pull/98/commits/06bc650f42c377ff5fdfe27a4a23af3efb7edb7f